src/scss/3_components/form/search-form.scss

```diff
@@ -0,0 +1,130 @@
+@import '../../init';
+
+#block-search-form,
+#block-swr-search {
+  overflow: visible;
+  position: relative;
+  width: 100%;
+  height: 3.5rem;
+  margin: 0 auto;
+  padding: 0;
+
+  @media screen and (min-width: $tab) {
+    margin-top: 30px;
+  }
+
+  .content {
+    margin-top: 0;
+  }
+
+  .form-text {
+    float: none;
+    width: 220px;
+    max-width: 100%;
+    margin-right: 0;
+    padding: 8px;
+  }
+
+  input:focus {
+    outline: none;
+  }
+
+  .form-submit:hover {
+    opacity: .8;
+  }
+}
+
+%search-form {
```

Comment:
Remove the silent placeholder, as it serves no purpose now that we creates singles css files. Also tends to bloat css, can amalgamate into single rules.